### PR TITLE
Fix for importing nodesets with different order

### PR DESCRIPTION
**Pull request by swamper123**
Made some changes to suppress migration, via optional parameter.

### CODE REVIEW

File: asyncua/common/xmlimporter.py [hunk 1]
```diff
@@ -31,7 +31,8 @@ async def _map_namespaces(self):
         xml_uris = self.parser.get_used_namespaces()
         server_uris = await self.server.get_namespace_array()
         namespaces_map = {}
-        for ns_index, ns_uri in enumerate(xml_uris):
+        xml_uris_enum = enumerate(xml_uris)
+        for ns_index, ns_uri in xml_uris_enum:
```
Comment (oroulet) [inline asyncua/common/xmlimporter.py:35]: that was a very strange change. I never saw an enumeration on two lines ;-)

File: asyncua/common/xmlimporter.py [hunk 2]
```diff
@@ -105,7 +106,7 @@ async def import_xml(self, xmlpath=None, xmlstring=None):
         nodes = []
         for nodedata in nodes_parsed:  # self.parser:
             try:
-                node = await self._add_node_data(nodedata)
+                node = await self._add_node_data(nodedata, suppress_migration=True)
```
Comment (oroulet) [inline asyncua/common/xmlimporter.py:109]: can'tyou call that argument: no_ns_migration or even no_namespace_migration that is easier to understand

File: asyncua/common/xmlimporter.py [hunk 3]
```diff
@@ -262,6 +268,25 @@ def _get_add_node_item(self, obj):
             node.TypeDefinition = self._migrate_ns(obj.typedef)
         return node
 
+    def _get_add_node_item_suppressed(self, obj):
```
Comment (oroulet) [inline asyncua/common/xmlimporter.py:271]: the work "suppressed" is also a bit stranged here. Not sure what you mean

**Comment by oroulet** [comment]
also it would be nice if you had a minimal example that has that issue, so you could add a test. I am really not sure what you are trying to fix

File: asyncua/common/xmlimporter.py (hunk 3 above)
Comment (swamper123) [reply asyncua/common/xmlimporter.py:271]: It should do the same as _get_add_note_item(), just without the migration, they should just take the obj.nodeid etc. But I will rework that.

**Comment by swamper123** [comment]
> so you could add a test

There is already an existing test for `test_xml_import_additional_ns` which is breaking...but I don't know why atm.

**Pull request merged**